distribution/lib/Standard/Base/0.0.0-dev/src/Function.enso

```diff
@@ -29,7 +30,8 @@ type Function
     <| : Any -> Any
     <| self ~argument = @Builtin_Method "Function.<|"
 
-    ## ADVANCED
```

Comment:
Do we need to hide `<<`? It can be quite useful and should show up only on `Function` anyway so should not be polluting the CB. Unless it was appearing elsewhere, then ignore me.